cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/MetricSampler.java

```diff
@@ -18,7 +18,7 @@
  * The interface to get metric samples of given topic partitions.
  * <p>
  * Kafka Cruise Control periodically collects the metrics of all the partitions in the cluster, including the leader and follower
- * replicas. The {@link #getSamples(Cluster, Set, long, long, SamplingMode, MetricDef, long)}
+ * replicas. The {@link #getSamples(MetricSamplerOptions)}
  * will be called for all the replicas of partitions in the cluster in one sampling period.
  * The MetricSampler may be used by multiple threads at the same time, so the implementation need to be thread safe.
  *
```

Comment:
(Not introduced in this patch) I noticed that this JavaDoc is not fully accurate. Can we update it to reflect that it not only collects partition metrics, but any `{@link CruiseControlMetric}`?